cli/jobs/automl-standalone-jobs/cli-automl-forecasting-orange-juice-sales/cli-automl-forecasting-orange-juice-sales.yml

```diff
@@ -0,0 +1,38 @@
+$schema: https://azuremlsdk2.blob.core.windows.net/preview/0.0.1/autoMLJob.schema.json
+type: automl
+
```

Comment:
We need to add the featurization configuration as it is done in V1 notebook

```
featurization:
  featurization_config:
    column_purposes:
      CPWVOL5: Numeric
    transformer_params:
      Imputer:
      - fields: ["Quantity"]
        parameters:
          strategy: constant
          fill_value: 0
      - fields: ["INCOME"]
        parameters:
          strategy: median
```

Reply:
@nick863 
I have added this, also added for field "Price"